nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/ui/provenance-event-listing/provenance-event-table/provenance-event-table.component.html

```diff
@@ -15,142 +15,184 @@
   ~ limitations under the License.
   -->
 
-<div class="provenance-event-table h-full flex flex-col">
-    <div class="flex flex-col">
-        <div class="value font-bold">Displaying {{ filteredCount }} of {{ totalCount }}</div>
-        <div class="flex justify-between">
-            <div>
-                Oldest event available: <span class="value">{{ oldestEventAvailable }}</span>
-            </div>
-            <div>
-                {{ resultsMessage }}
-                <a *ngIf="hasRequest" (click)="clearRequestClicked()">Clear Search</a>
+<div class="provenance-event-table h-full">
+    <div [class.hidden]="showLineage" class="h-full flex flex-col gap-y-2">
+        <div class="flex flex-col">
+            <div class="value font-bold">Displaying {{ filteredCount }} of {{ totalCount }}</div>
```

Comment:
Consider hiding this unless there is a filter active. It can actually be a bit confusing now that there is pagination. It tells us that 1000 of 1000 are displayed, but with pagination we only ever show 100 so it can be a bit confusing.
Might also consider re-wording it so it is clear that it is referring to the number of items that pass the filter, but not necessarily how many are displayed.

Reply:
Good call. Will update.